docs/mkdocs.yml

```diff
@@ -1,58 +1,94 @@
+%YAML 1.2
+---
 site_name: LSP for Sublime Text
+site_description: Client implementation of the Language Server Protocol for Sublime Text
+site_author: Sublime Text user community
 site_url: https://sublimelsp.github.io/LSP/
+
+repo_name: sublimelsp/LSP
 repo_url: https://github.com/sublimelsp/LSP/
-edit_uri: edit/main/docs/src/
-docs_dir: 'src'
+edit_uri: blob/main/docs/src/markdown
+
+docs_dir: 'src/markdown'
 
 plugins:
-  - search
-  - redirects:
-      redirect_maps:
-        'guides/cplusplus.md': 'https://github.com/sublimelsp/LSP-clangd'
-        'guides/client_configuration.md': 'client_configuration.md'
```

Comment:
Can we keep this? It's for handling old links that might exist in various places.